extensions/sql-database-projects/src/dialogs/addSqlBindingQuickpick.ts

```diff
@@ -204,7 +224,31 @@ export async function launchAddSqlBindingQuickpick(uri: vscode.Uri | undefined,
 					}
 					if (connectionString) {
 						try {
-							const success = await azureFunctionsUtils.setLocalAppSetting(path.dirname(projectUri.fsPath), newConnectionStringSettingName, connectionString);
+							const projectFolder: string = path.dirname(projectUri.fsPath);
+							const localSettingsPath: string = path.join(projectFolder, constants.azureFunctionLocalSettingsFileName);
+							// Connection string does not include a password, so will prompt user to optionally enter it manually
+							if (!connectionInfo?.password || includePassword === constants.noString && connectionInfo.authenticationType !== 'SqlLogin') {
+								let userPassword = await vscode.window.showInputBox({
+									prompt: constants.enterPasswordPrompt,
+									placeHolder: constants.enterPasswordManually,
+									ignoreFocusOut: true,
+									password: true,
+									validateInput: input => input ? undefined : constants.valueMustNotBeEmpty
+								});
+								if (!userPassword) {
+									// User will have to manually enter password later
+									void vscode.window.showWarningMessage(constants.userPasswordLater, constants.openFile, constants.closeButton).then(async (result) => {
+										if (result === constants.openFile) {
+											// open local.settings.json file
+											void vscode.commands.executeCommand(constants.vscodeOpenCommand, vscode.Uri.file(localSettingsPath));
+										}
+									});
+								} else {
+									// replace the default password with the user's password
+									connectionString = connectionString.replace(constants.passwordPlaceholder, userPassword);
```

Comment:
Are we guaranteed to always have the placeholder? What happens if `includePassword` is true but the password isn't saved? Or does the connection info have it from the node when the user connected?

Reply:
Yes it should always be a placeholder. IncludePassword cannot be true since we only ask that prompt if the password is saved in the first place (via the connectionInfo)